e2e/steps_file.js

```diff
@@ -26,5 +26,12 @@ module.exports = function () {
     seeEventSubmissionConfirmation(event) {
       this.see(`updated with event: ${event}`);
     },
+
+    clickHyperlinkAndReturnToOriginalPage(link, pageGoingTo, originalPage) {
```

Comment:
I would probably remove instructions that take users back from this function. This is to keep functions single responsibility.

Reply:
Fine by me